### PR TITLE
Gradle Enterprise plugin 3.3-rc-1

### PR DESCRIPTION
**Pull request by guylabs**
@wolfs Should we remove the verification entries for old Gradle Enterprise plugins in the `verification-metadata.xml` file?

### CODE REVIEW

**Comment by wolfs** [comment]
@guylabs I think we shouldn't.

File: gradle/init-scripts/build-scan.init.gradle.kts [hunk 1]
```diff
@@ -29,6 +31,10 @@ fun configureExtension(extension: Any) {
         }
         setProperty("server", "https://e.grdev.net")
 
+        if (ci) {
+            setProperty("uploadInBackground", false)
```
Comment (jjohannes) [inline gradle/init-scripts/build-scan.init.gradle.kts:35]: :o: Why is this required if this is already done in BuildScanPlugin.kt ?

Comment (guylabs) [reply gradle/init-scripts/build-scan.init.gradle.kts]: I wasn't sure how these were used that is why I added it on both places. Removed it.
